azure-kusto-data/tests/aio/test_async_token_providers.py

```diff
@@ -162,8 +161,8 @@ async def test_msi_provider(self):
             print(" *** Skipped MSI Provider Test ***")
             return
 
-        user_msi_object_id = os.environ.get("MSI_OBJECT_ID")
-        user_msi_client_id = os.environ.get("MSI_CLIENT_ID")
+        user_msi_object_id = get_env("MSI_OBJECT_ID")
+        user_msi_client_id = get_env("MSI_CLIENT_ID")
```

Comment:
So this throws now?
Why?

Reply:
I thought that because the test is under an env variable these are required - my mistake

Reply:
Fixed